tests/HTMLPurifierTest.hack

```diff
@@ -119,12 +119,19 @@ class HTMLPurifierTest extends HackTest {
 </table>';
 		$purifier = new HTMLPurifier\HTMLPurifier($config);
 		$clean_html = $purifier->purify($dirty_html);
-		expect($clean_html)->toEqual('<table><caption>
+		expect($clean_html)->toEqual('<table>
```

Comment:
This test failed because of indentation. I am not sure why travis shows test pass even though on local test actually failed.
Same for the following test.

Reply:
This is the issue where spacing requirements on Mac vs Linux are different for still unidentified reasons.  I think we'll have to revert this change to get the CI tests to pass.